lib/services/auth-service.js

```diff
@@ -67,6 +58,24 @@ function authServiceFactory(
             secretOrKey : self.secretOrKey,
             ignoreExpiration : self.ignoreTokenExpiration
         };
+
+        // used to serialize the user for the session
```

Comment:
@zyoung51 I'm a little confused by this code.  My understanding is that serializeUser produces a unique value that can be used by the deserializeUser function to reference a user object.  This value then gets stored in the session.  In this case, it looks like you're serializing the whole user object to the session, but then expecting a user id in the deserializeUser function.  The id passed to deserializeUser will be the whole user object so I would think you could just return done(null, user).  Maybe this is working differently because we're not using sessions?

The example from the passport documentation looks like this:
```javascript
passport.serializeUser(function(user, done) {
  done(null, user.id);
});

passport.deserializeUser(function(id, done) {
  User.findById(id, function(err, user) {
    done(err, user);
  });
});
```